docs/DeveloperGuide.md

```diff
@@ -238,13 +240,13 @@ The following activity diagram summarizes what happens when a user executes a ne
     * Cons: Can be very messy to implement.
 
 
-### \[Proposed\] Data archiving
+### Data archiving
```

Comment:
```suggestion
### Data import and export
```
Think "archiving" is a bit different from "saving offline"—with "archiving" I would expect archived cases to still be accessible somewhere in the app?